packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -1,15 +1,30 @@
 import { WalletConfig } from '../wallets.constants';
 import { WalletPage } from '../wallet.page';
 import { expect } from '@playwright/test';
-import { test, BrowserContext, Page, Locator } from '@playwright/test';
+import { test, BrowserContext, Page } from '@playwright/test';
+import { MainPage, LoginPage, SettingsPage } from './pages';
+import {
+  OnboardingElement,
+  OperationActions,
+  Header,
+  NetworkList,
+  OptionsMenu,
+  PopoverElements,
+  AccountMenu,
+} from './pages/elements';
 
 export class MetamaskPage implements WalletPage {
```

Comment:
maybe we should name MetamaskApp for now?

Reply:
agree

Reply:
agree